tests/requirement/test_requirement_loader.py

```
@@ -0,0 +1,42 @@
+from django.test import TestCase
+from logging import getLogger
+
+from endoreg_db.models import (
+    Requirement, 
+    PatientDisease, # Added PatientDisease
```

Ruff (0.11.9)

`endoreg_db.models.PatientDisease` imported but unused

Remove unused import

(F401)